src/ts/types/__tests__/index.ts

```diff
@@ -87,6 +87,13 @@ export const events = async () => {
             payload.features.unfinished_backup;
             payload.features.no_backup;
             payload.features.model;
+            // @ts-ignore: error does not exist
+            payload.error.toLowerCase();
```

Comment:
maybe this would work in TS file?
```suggestions
payload.error?.toLowerCase();
```

Reply:
`@ts-expect-error` it is